extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1521,6 +1521,17 @@ namespace Microsoft.Windows.DevHome.SDK
         };
     }
 
+    // Used by the extension to provide information on the chat style generation
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 6)]
+    runtimeclass QuickStartChatStyleResult
+    {
```

Comment:
Do we think we want a class to provide flexibility for the chat result for the future rather than just using string below in the async operation?